### DISCUSSION TITLE
help please no integration

### DISCUSSION THREAD

**Discussion by Nasdrovia69** [gsamokovarov/jump#85] [Q&A]
Hello
I like your program from what little I've seen of it, but I can't get past the integration stage.
I'm on Fedora and I did put 
 eval "$(jump shell bash)" once and 
eval "$(jump shell)" once in the .bash_profile and once in the .bashrc.
but it doesn't work, it doesn't generate the lines of code you're describing. I don't understand something. Sorry if I'm asking for your help, but could you point me in the right direction?
Sorry for my english
Translated with DeepL.com (free version)

**Comment by gsamokovarov** [comment]
Sorry for seeing your answer so late. Could you let me know if you managed to integrate it? You can `echo $SHELL` to see what shell you are running.
